.gitignore

```diff
@@ -2,7 +2,6 @@
 deploy
 .import/
 *.translation
-export_presets.cfg
```

Comment:
This was excluded per this suggestions:
https://github.com/godotengine/godot-demo-projects/issues/329

https://docs.godotengine.org/en/stable/tutorials/best_practices/version_control_systems.html#files-to-exclude-from-vcs

Overall I think we can actually add that, but we need to remember not to leak credentials locally. On GitHub we can use secrets, though I don't know if we're able to add Android seamlessly with just the  actions you've used. We'll think later I guess